src/actions/mediaModalActions.ts

```diff
@@ -0,0 +1,15 @@
+import { MEDIA_MODAL_OPEN, MEDIA_MODAL_CLOSED } from "../constants/mediaModalConstants";
```

Comment:
try running `yarn lint` to fix all the double quotes to single. Not sure which code editor/idea you are using, but I think VSCode requires the ESLint plugin for errors to show up in the editor console, and WebStorm needs ESLint to be enabled if it's not already.

Reply:
ah I'm using Webstorm... I should enable that :P